monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -105,6 +105,15 @@ include::modules/monitoring-modifying-retention-time-for-prometheus-metrics-data
 * xref:../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[Understanding persistent storage]
 * xref:../scalability_and_performance/optimizing-storage.adoc#optimizing-storage[Optimizing storage]
 
+// Configuring remote write storage for Prometheus
+include::modules/monitoring-configuring-remote-write.adoc[leveloffset=+1]
+
+.Additional resources
+
```

Comment:
Generally we need PX team acks for Additional resources in the docs. @sferich888 are we good with these links?

Reply:
@sferich888 Are we good with the links in this PR? Thanks.